### DISCUSSION TITLE
Viewing currently active authorization

### DISCUSSION THREAD

**Discussion by brolifen** [lithnet/access-manager#188] [Ideas]
Would be cool if the web app also had a section to show currently active JIT authorization. Perhaps with an "extend" button if expiry is reaching soon.

**Comment by ryannewington** [comment]
Great idea. Our next version will have some major uplift done to the web ui, so this is something we can consider adding in for sure.

**Reply by dnauman1** [reply]
I saw this comment and wanted to follow up when the next version may be released? The last feature update looks like was back in May for Windows LAPS.
